vendor/github.com/openshift/api/console/v1/0000_10_consoleyamlsample.crd.yaml

```diff
@@ -54,6 +54,11 @@ spec:
               description: description of the YAML sample.
               type: string
               pattern: ^(.|\s)*\S(.|\s)*$
+            snippet:
```

Comment:
@andrewballantyne @christianvogt this bumps the console operator for the `snippet` change